Examples/clim-fig.lisp

```diff
@@ -427,7 +427,8 @@
       (T
        (erase-output-record latest-undo-entry *standard-output*)
        (push latest-undo-entry (clim-fig-redo-list *application-frame*))
-       (activate-gadget (find-pane-named *application-frame* 'clear))))
+       (activate-gadget (find-pane-named *application-frame* 'clear))
+       (activate-gadget (find-pane-named *application-frame* 'redo))))
```

Comment:
I have changes that synchronize the active/inactive state of these buttons with the corresponding commands (including menus). Should I submit those as a pull request?

Reply:
Sure, go ahead.